dump27/claims_max/text.py

```
@@ -189,14 +222,15 @@ def make_section(pid, table, old_data, max_n=51):
             continue
         # ---
         old_v = old_data_qids.get(qid, 0)
+        # ---
         diffo = min_it(count, old_v, add_plus=True)
         # ---
         table_rows.append(f"! {idx} \n| {{{{Q|{qid}}}}} \n| {count:,} \n| {diffo}")
     # ---
     old_others = old_data_qids.get("others", 0)
     diff_others = min_it(other_count, old_others, add_plus=True)
```

Ruff (0.8.2)

Local variable `diff_others` is assigned to but never used

Remove assignment to unused variable `diff_others`

(F841)